glean-core/preview/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "glean-preview"
+version = "0.0.1"
+authors = ["Jan-Erik Rediger <jrediger@mozilla.com>", "The Glean Team <glean-team@mozilla.com>"]
+description = "Nice Glean Rust API"
+repository = "https://github.com/mozilla/glean"
+readme = "README.md"
+license = "MPL-2.0"
+edition = "2018"
+keywords = ["telemetry"]
+include = [
+  "README.md",
+  "LICENSE",
+  "src/**/*",
+  "tests/**/*",
+  "Cargo.toml",
+]
+
+[badges]
+circle-ci = { repository = "mozilla/glean", branch = "master" }
+maintenance = { status = "actively-developed" }
+
+[dependencies.glean-ffi]
+path = "../ffi"
+version = "21.2.0"
```

Comment:
```suggestion
version = "21.3.0"
```

Reply:
Fun fact: it actually pulls in 21.3.0 ;)